src/style.css

```
@@ -0,0 +1,126 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  background-color: #e5e5e5;
+}
+
+.card {
+  background: #fff;
+  height: 406px;
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+}
+
+.card__info {
+  display: flex;
+  flex-direction: column;
+}
+
+.info__image {
+  margin: 32px 19px 0;
+  height: 134px;
+  width: 160px;
+}
+
+.info__name {
+  height: 36px;
+  width: 166px;
+  margin: 40px 16px 0;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.info__code {
+  margin: 4px 16px 0;
+  font-weight: 400;
+  font-size: 10px;
+}
+
+.info__review {
+  display: flex;
+  height: 16px;
+  justify-content: space-between;
+  align-items: center;
+  margin: 17px 16px 0;
+}
+
+.review__stars {
+  display: flex;
+  justify-content: space-around;
+  height: 16px;
+  margin-left: 2px;
+  /* gap: 4px; */
+}
+
+.stars__star {
+  background: url("./images/star-active.svg") no-repeat;
+  width: 16px;
+  margin: 2px 0 0;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  background: url("./images/star.svg") no-repeat;
+}
+
+.review__count {
+  font-weight: 400;
+  font-size: 10px;
+}
+
+.info__price {
+  display: flex;
+  height: 18px;
+  justify-content: space-between;
+  align-items: center;
+  margin: 24px 16px 16px;
+
+}
+
+.price__text {
+  font-weight: 400;
+  font-size: 12px;
+}
+
+.price__value {
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.info__buttom {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-transform: uppercase;
+  margin: 0 16px;
+  height: 40px;
+  text-decoration: none;
+  background-color: #00acdc;
+  font-weight: 700;
+  font-size: 14px;
+  color: white;
```

```suggestion
  color: #fff;
```
краще використовувати шістнадцяткові вирази для кольорів